src/ext-list.js

```diff
@@ -38,6 +38,7 @@ export async function updateList(exts = extensions, el = document.getElementById
       </summary>
       <div class='requests'>
       ${Object.keys(ext.reqUrls).map(url => `
+        <div class='item-block pointer ${ext.reqUrls[url].isBlocked ? ' blocked' : ''}' id='block-ext-url-${url}|${ext.id}' title='${ext.reqUrls[url].isBlocked ? 'Unblock' : 'Block'} network requests for this url'></div>
```

Comment:
I think there were some restrictions on what characters are allowed in the HTML `id` attribute, not sure if some URL's would work as `id`

Reply:
please check https://github.com/dnakov/little-rat/pull/18/commits/215586cedea8209c97be636366a890824be11b2e this commit.

Reply:
That looks good, I think the URL is already encoded, so there won't be spaces in it.